src/components/navMenu/mobileNavMenu.tsx

```diff
@@ -0,0 +1,101 @@
+import { dcNavLinks } from "@/data/dcNavLinks";
+import {
+  Text,
+  Link,
+  Icon,
+  Button,
+  useCloseDropDown,
+  List,
+} from "@nypl/design-system-react-components";
+import React, { useRef, useState } from "react";
+import FocusLock from "@chakra-ui/focus-lock";
+import { Box, chakra } from "@chakra-ui/react";
+import { useScrolled } from "@/hooks/useScrolled";
+
+const listItems = dcNavLinks.map(({ href, text }) => (
+  <Link
+    isUnderlined={false}
+    href={href}
+    aria-label={text}
+    key={text}
+    sx={{
+      display: "block",
+      paddingBottom: "m",
+      paddingLeft: "16px",
+      width: "100%",
+      ":hover": {
+        textDecoration: "none",
+      },
+      ":visited": {
+        color: "ui.black",
+      },
+    }}
+  >
+    <Text size="subtitle2" sx={{ marginBottom: "unset" }}>
```

Comment:
`p` elements should not render inside `a` elements so you can remove the `Text` component and move the styles to the `Link` component.